### PR TITLE
Removed challenging estimateAmountOut function

### PR DESCRIPTION
**Pull request by trmid**
```
Running 2 tests for test/fuzz/LiquidationPairFuzz.t.sol:LiquidationPairFuzzTest
[FAIL. Reason: Assertion failed. Counterexample: calldata=0xc5ac26850000000000000000000000000000000000000000000000000000013bc5a73f9300000000000000000000000000000000000000000000000000000000000d79da, args=[1356230770579 [1.356e12], 883162 [8.831e5]]] testEstimateAmountOut(uint104,uint32) (runs: 152, μ: 37704, ~: 27028)
[PASS] testSwapMaxAmountOut(uint104,uint32) (runs: 256, μ: 55558, ~: 64715)
Test result: FAILED. 1 passed; 1 failed; 0 skipped; finished in 112.46ms
```

### CODE REVIEW

File: src/libraries/ContinuousGDA.sol [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: GPL-3.0
 pragma solidity 0.8.19;
 
+import "forge-std/console2.sol";
+
```
Comment (trmid) [inline src/libraries/ContinuousGDA.sol]:
```suggestion
```

File: test/fuzz/LiquidationPairFuzz.t.sol [hunk 2]
```diff
@@ -23,6 +23,8 @@ contract LiquidationPairFuzzTest is Test {
     uint104 initialAmountOut = 1e18;
     uint256 minimumAuctionAmount = 2e18;
 
+    SD59x18 percentDeviation = wrap(0.01e18);
```
Comment (PierrickGT) [inline test/fuzz/LiquidationPairFuzz.t.sol]: Why is this variable needed now?

File: test/fuzz/LiquidationPairFuzz.t.sol [hunk 3]
```diff
@@ -23,6 +23,8 @@ contract LiquidationPairFuzzTest is Test {
...
+
```
Comment (trmid) [inline test/fuzz/LiquidationPairFuzz.t.sol]:
```suggestion
```

File: test/fuzz/LiquidationPairFuzz.t.sol (hunk 2 above)
Comment (trmid) [reply test/fuzz/LiquidationPairFuzz.t.sol]: looks like it isn't used anywhere so it's probably left over from debugging

**Pull request merged**